Userland/Libraries/LibWeb/UIEvents/WheelEvent.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022, the SerenityOS developers.
```

Comment:
It's fine to use the generic copyright, but contributors are encouraged to add their own attribution